### DISCUSSION TITLE
How to pair Heiman HS3SA with zigbee2MQTT

### DISCUSSION THREAD

**Discussion by metu71** [Koenkk/zigbee2mqtt#7994] [Devices]
Hello,

I'm just starting my journey with zigbee and got myself 5 of Heiman HS3SA smoke detectors. I just can't get them to pair...

Does anyone have these in use and how did you manage to pair them with zigbee2mqtt?

**Comment by ovizii** [comment]
Same problem here, there are no instructions: https://www.zigbee2mqtt.io/devices/HS3SA.html

**Comment by VincentSC** [comment]
New link is here: https://www.zigbee2mqtt.io/devices/HS3SA_HS1SA.html

To pair, first reset by long-pressing about 6 seconds, then pair by long-pressing 2 seconds. After a reset there is slow blinking, when pairing there is fast-blinking.
